README.md

```diff
@@ -39,6 +59,22 @@ Run this command in the root of the project directory to install NimblePhxGenTem
 mix do deps.get, deps.compile
 ```
 
+Step 4: Up and running the Phoenix 
+
+Run this command and verify your local website is successfully installed.
+
+```bash
+mix do deps.get, deps.compile
+```
+
+Step 5:
+
+After the project initialized and ran successfully, please remove `nimble_phx_gen_template` in the `mix.exs` file. Then run this command:
```

Comment:
Does it make sense to add this step into the `## Installation` session 🤔

What do you think if we combine the `## Installation` and `## Usage` in to 1 section @byhbt 

```
Step 1
Step 2
Step 3
Step 4 # usage
```
```
mix nimble.phx.gen.template -v      # Print the version
mix nimble.phx.gen.template --web   # Apply the Web template
mix nimble.phx.gen.template --api   # Apply the API template
mix nimble.phx.gen.template --live  # Apply the LiveView template
```
```
Step 5 # remove
```

Reply:
But do you think we could remove the `start Phoenix` step first @byhbt 🤔 

If we keep these section separately then not sure it doesn't make the Installation clearly because it contain the uninstall right away after mix deps.get...

Reply:
1. Removed the Start Phoenix step
2. I also moved the section remove dependencies to the `Note ` section after the user generate the project in the `Usage` section.